make: packageqa fixes

sknsean opened the pull request:
Signed-off-by: Sean Nyekjaer <sean.nyekjaer@prevas.dk>

esben commented on recipes/make/make.inc:
```diff
@@ -12,6 +12,9 @@ RECIPE_TYPES = "native sdk"
 DEPENDS = ""
 
+DEPENDS_${PN} = "libc"
+RDEPENDS_${PN}:sdk = "libc"
```
Why the ':sdk' override?

sknsean commented:
Oh at first I did write RDEPENDS = "libc" and it complained, so i added the :sdk.
Later i added the _${PN} so i guess the :sdk is not needed then...
